core/common/src/rpc/web/WebAppRpcLogging.ts

```diff
@@ -54,22 +54,22 @@ export class WebAppRpcLogging {
   }
 
   private static findPathIds(path: string) {
-    // SWB
-    let contextId = "";
+    let iTwinId = "";
     let iModelId = "";
 
     const tokens = path.split("/");
     for (let i = 0; i !== tokens.length; ++i) {
+      // SWB How is this regex being used, what else needs to be renamed to match it?
```

Comment:
This is based on the usage within the Rpc mechanism itself which is still set as context to not break our service contracts. If we did make any update here we should look for both context and itwin.

Reply:
Added an additional search for "itwin" as well.